MicrosoftFluentUI.podspec

```diff
@@ -284,9 +290,16 @@ Pod::Spec.new do |s|
   s.subspec 'DatePicker_mac' do |datepicker_mac|
     datepicker_mac.platform = :osx
     datepicker_mac.dependency 'MicrosoftFluentUI/Core_mac'
+    datepicker_mac.dependency 'MicrosoftFluentUI/DynamicColor_mac'
```

Comment:
Shouldn't this be MicrosoftFluentUI/Appearance_mac?
DatePicker does not depend on DynamicColor or ColorSet, correct?

Reply:
yes, that's correct.